src/sync/sync_session.cpp

```diff
@@ -211,15 +211,7 @@ struct sync_session_states::Dying : public SyncSession::State {
     void enter_state(std::unique_lock<std::mutex>&, SyncSession& session) const override
     {
         size_t current_death_count = ++session.m_death_count;
-        session.m_session->async_wait_for_upload_completion([session=&session, current_death_count](std::error_code error_code) {
-            if (error_code == util::error::operation_aborted) {
```

Comment:
For example, in the case where the Client is stopped but the sessions were in the inactive sessions list? Okay, that makes sense.